server/chat-plugins/rock-paper-scissors.ts

```diff
@@ -0,0 +1,406 @@
+/**
+ * Rock, Paper, Scissors plugin by Mia
+ * @author mia-pi-git
+ */
+import {Utils} from '../../lib/utils';
+
+/** Map<to, [from, rounds]> */
+const challengeRequests: Map<string, string> = new Map();
+
+const MAX_ROUNDS = 500;
+const TIMEOUT = 10 * 1000;
+const ICONS: {[k: string]: string} = {
+	Rock: `<i class="fa fa-hand-rock-o"></i>`,
+	Paper: '<i class="fa fa-hand-paper-o"></i>',
+	Scissors: '<i class="fa fa-hand-scissors-o"></i>',
+};
+
+export class RPSPlayer extends Rooms.RoomGamePlayer {
+	user: User;
+	currentChoice: string;
+	name: string;
+	constructor(user: User, game: RPSGame) {
+		super(user, game);
+		this.user = user;
+		this.name = user.name;
+		this.currentChoice = "";
+	}
+}
```

Comment:
```suggestion
export class RPSPlayer extends Rooms.RoomGamePlayer {
	currentChoice = '';
}
```

We intentionally don't store `user`, and none of the rest of this code does anything.